README.md

```diff
@@ -180,3 +180,8 @@ performanceChecksCallback | Boolean | Gets called for every failed performanceCh
 warningsEnabled | Boolean | Will notify you about library misuse and invalid/incompatible options. (Barely implemented, mostly to-do.)
 warningsCallback | Function | Gets called for every warning item; this is `console.warn` by default.
 exceptionCallback | Function | Gets called if your selector function throws an exception.
+onInvoke | Function | Callback fired whenever the selector is executed. Useful for debugging.
+onSkippedRun | Function | Callback fired when a selector returns its cached value directly. Useful for debugging.
+onPhantomRun | Function | Callback fired when a selector runs but returns something equivalent to its cached value. Useful for debugging.
```

Comment:
I like this. I would like to be able to use this to see if a parent selector isn't optimized as it should be 👍